### PR TITLE
Thunderscope: Draw orientation robots

### LINKED ISSUE #2397: Thunderscope: Draw orientation of the robots
### Description of the task

Currently, in thunderscope, the robots are perfectly round, but in reality, they have a flat spot where we kick/chip and dribble from. We want to draw the flat spot, _or_, have some way of showing which way they are facing. Our jupyter notebook indicates orientation with a line.

What thunderscope currently looks like:
image

What it could look like:
image

### Acceptance criteria

- [ ] Modify the draw_team function in `software/thunderscope/world_layer.py` to draw the robot orientation
- [ ] Draw the robot as an arc + line, or draw it as a circle + line
- [ ] Test manually

### Blocked By
#2394

### PR DESCRIPTION
**Pull request by piramonpan**
### Description

Draw Orientation of the robots by changing the their shape from circle to an arc + line, indicating which direction the robots are facing 

### Testing Done

Manual Testing

### Resolved Issues

resolves #2397  

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/thunderscope/field/world_layer.py [hunk 1]
```diff
@@ -86,12 +87,15 @@ def draw_team(self, painter, color, team: Team):
 
             # TODO (#2396) Draw the robot IDs of the robots
             # TODO (#2397) Draw the Orientation of the robots
-            painter.drawEllipse(
+            painter.drawChord(
```
Comment (jonathanlew) [inline src/software/thunderscope/field/world_layer.py]: can we delete this TODO about orientation?

File: src/software/thunderscope/field/world_layer.py [hunk 2]
```diff
@@ -86,12 +87,15 @@ def draw_team(self, painter, color, team: Team):
...
                 self.createCircle(
                     robot.current_state.global_position.x_meters * MM_PER_M,
                     robot.current_state.global_position.y_meters * MM_PER_M,
                     ROBOT_MAX_RADIUS,
-                )
+                ),
+                (math.degrees(robot.current_state.global_orientation.radians) + 45)
+                * -16,
+                270 * -16,
```
Comment (jonathanlew) [inline src/software/thunderscope/field/world_layer.py]: nit; can we pull this -16 out into a variable

**Review by jonathanlew — COMMENTED** [review]
looks great!

**Comment by akhilveeraghanta** [comment]
nothing to add beyond whats already been said, nice work!!

File: src/software/thunderscope/field/world_layer.py (hunk 2 above)
Comment (piramonpan) [reply src/software/thunderscope/field/world_layer.py]: yeah sure! Should I name the variable CONVERT_DEGREE and put it in software/thunderscope/constants.py folder?

Comment (jonathanlew) [reply src/software/thunderscope/field/world_layer.py]: just naming a variable in this function is fine

**Review by jonathanlew — APPROVED** [review]
LGTM!

**Review by akhilveeraghanta — APPROVED** [review]
lgtm! :100:

**Pull request merged**